test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -4828,7 +4838,9 @@ BOOST_AUTO_TEST_CASE(warn_about_callcode)
 			}
 		}
 	)";
-	CHECK_WARNING(text, "\"callcode\" has been deprecated in favour of \"delegatecall\"");
```

Comment:
You can replace `var` with `bool` in this example.

Reply:
Changing `var` to `bool` here gives me the message 

> `Type function () payable returns (bool) is not implicitly convertible to expected type bool.`

Reply:
You might be able to go with just `function f() pure public { address(0x12).callcode; }` for now.

Reply:
OK, then probably best to just remove the variable.